src/gmail.js

```diff
@@ -1759,7 +1759,7 @@ var Gmail_ = function(localJQuery) {
 
 
     api.helper.get.is_delegated_inbox = function() {
-        return api.tracker.globals[17][5][0] === "acfg";
+        return $(".identityUserDelegatedAccount").length === 1;
     };
```

Comment:
Are we sure this will always be Length one? Are there cases you can see where we may have *more* than one?

Wouldn't it be better/more resilient to change to do a check if it's not zero?

Reply:
It is always 0 or 1 from my testings (on the different accounts / contexts listed in the spreadsheet), and it should always be as this selector identifies the logged in account in the accounts list when connected in a delegated inbox, so 1 max at a time.

I would say it is more a matter of "preference", either we prefer that this check fails if this css class starts appearing somewhere else in Gmail as it is not expected or we consider that if it appears more than once we are probably still in a delegated inbox context.

Reply:
In that case, looks good to me.

If nobody else has any objections over the following days, I will merge this as is.

Thanks for the patch!